catalog_classes/item.rb

```diff
@@ -11,7 +11,7 @@ def initialize(genre, author, source, label, publish_date)
   end
 
   def can_be_archived?
-    published_date > 10
+    (Time.now.year - @publish_date.year) > 10
```

Comment:
- [ ] This line threw an error. I think this is because you didn't parse `@publish_date` as date before using the Date method  `year` `@publish_date.year`